internal/database/migrations/000001_create_entities_table.up.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE positions (
+                           id VARCHAR PRIMARY KEY,
+                           name VARCHAR(255) NOT NULL,
+                           salary INT NOT NULL,
+                           created_at TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP,
+                           updated_at TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP
+);
+
+CREATE TABLE employees (
+                           id VARCHAR PRIMARY KEY,
+                           first_name VARCHAR(255) NOT NULL,
+                           last_name VARCHAR(255) NOT NULL,
+                           position_id VARCHAR REFERENCES positions(id),
+                           created_at TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP,
+                           updated_at TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP
+);
+
+CREATE INDEX idx_employees_first_name ON employees(first_name);
+CREATE INDEX idx_employees_last_name ON employees(last_name);
+CREATE INDEX idx_employees_first_last_name ON employees(first_name, last_name);
```

Comment:
Are you sure you need all this indexes?

Reply:
> DB has to execute high efficiency search by FirstName, LastName and both combinations of FirstName and LastTame.
 that's how i understand this subtask, then i don't need any indexes?

Reply:
I don't think you need one, now. Indexes should be used when they needed, but also in some cases they might be suboptimal.

You should try experimenting with them using EXPLAIN:
https://www.postgresql.org/docs/current/sql-explain.html
https://www.postgresql.org/docs/current/using-explain.html